src/isIsogram.test.js

```diff
@@ -6,4 +6,24 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('no repeating letter', () => {
```

Comment:
It's a good practice to make your test case descriptions more descriptive. Currently, all your test cases have the same description 'no repeating letter' which doesn't provide much information about what each test case is testing. Consider including more details in your descriptions, for example, 'should return false for a word with repeating letters' or 'should return true for a word without repeating letters'.